superset/models/dashboard.py

```diff
@@ -361,15 +364,16 @@ def export_dashboards(  # pylint: disable=too-many-locals
     @classmethod
     def get(cls, id_or_slug: str) -> Dashboard:
         session = db.session()
```

Comment:
Unrelated, but seems strange that this is creating a new session here

Reply:
It does. I didn't feel entirely comfortable removing it though. Not sure why it was put there.